app/controllers/profiles_controller.rb

```diff
@@ -3,11 +3,7 @@
 # API for Profiles
 class ProfilesController < ApplicationController
   def index
-    profiles = scope_search.sort_by(&:score)
-    render json: ProfileSerializer.new(
-      profiles,
-      metadata(total: profiles.count)
-    )
+    render json: ProfileSerializer.new(scope_search.sort_by(&:score), metadata)
```

Comment:
This and other controllers that are not `rule_results` had associated tests that would not work if I did not adapt them to the new changes.